docs/source/about/advanced_usage.rst

```diff
@@ -752,6 +752,40 @@ hevc_mode
 
       hevc_mode = 2
 
+capture
+^^^^^^^
+
+**Description**
+   Force specific screen capture method.
+
+   .. Caution:: Applies to Linux only.
+
+**Choices**
+
+.. table::
+   :widths: auto
+
+   =========  ===========
+   Value      Description
+   =========  ===========
+   nvfbc      Use NVIDIA Frame Buffer Capture to capture direct to GPU memory. This is usually the fastest method for
+              NVIDIA cards. For GeForce cards it will only work with drivers patched with
+              `nvidia-patch <https://github.com/keylase/nvidia-patch/>`_
+              or `nvlax <https://github.com/keylase/nvidia-patch/>`_.
```

Comment:
Was this intentional to point to the keylase patch? Someone brought this up on discord.

Reply:
No it was supposed to be https://github.com/illnyang/nvlax

Reply:
Thanks, submitted #1205 to correct it.